app/actions/local/thread.ts

```diff
@@ -1,16 +1,20 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {General, Screens} from '@constants';
+import {updateThreadRead} from '@actions/remote/thread';
```

Comment:
there should not be remote actions in local actions.!!!

Reply:
Moving it to remote! 🙂  .. I have seen few actions in local pointing to remote, thought it was okay to have them.